api/spaces/spaces.go

```diff
@@ -175,3 +175,66 @@ func (api *API) RenameSpace(oldName string, newName string) error {
 	}
 	return nil
 }
+
+// RemoveSpace removes a space.
+func (api *API) RemoveSpace(name string, model string, force bool, dryRun bool) (network.RemoveSpace, error) {
+	var response params.RemoveSpaceResults
+	args := params.RemoveSpaceParams{
+		SpaceParams: []params.RemoveSpaceParam{{
+			Space:  params.Entity{Tag: names.NewSpaceTag(name).String()},
+			Force:  force,
+			DryRun: dryRun,
+		}},
+	}
+	err := api.facade.FacadeCall("RemoveSpace", args, &response)
+	if err != nil {
+		if params.IsCodeNotSupported(err) {
+			return network.RemoveSpace{}, errors.NewNotSupported(nil, err.Error())
+		}
+		return network.RemoveSpace{}, errors.Trace(err)
+	}
+	if len(response.Results) == 0 {
+		return network.RemoveSpace{}, nil
+	}
```

Comment:
may want to add an additional check in case the returned values are longer than 1